### PR TITLE
awscli: 1.19.20 -> 1.19.22

### PR DESCRIPTION
**Pull request by NeQuissimus**
###### Motivation for this change
Updates

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rmcgibbo** [comment]
LGTM (for some reason @r-rmcgibbo crashed, totally unrelated to the PR)

----------------

Result of `nixpkgs-review pr 115431` at 85441913 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>awsebcli</li>
    <li>mlflow-server</li>
    <li>python38Packages.flower</li>
    <li>python38Packages.geopandas</li>
    <li>python38Packages.osmnx</li>
    <li>python39Packages.flower</li>
    <li>python39Packages.geopandas</li>
    <li>python39Packages.osmnx</li>
  </ul>
</details>
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>dvc-with-remotes</li>
    <li>python38Packages.cartopy</li>
    <li>python38Packages.gradient</li>
    <li>python38Packages.snowflake-connector-python</li>
    <li>python38Packages.snowflake-sqlalchemy</li>
    <li>python39Packages.cartopy</li>
  </ul>
</details>
<details>
  <summary>11 packages skipped</summary>
  <ul>
  </ul>
</details>
<details>
  <summary>106 packages built:</summary>
  <ul>
    <li>aws-adfs (python38Packages.aws-adfs)</li>
    <li>aws-google-auth</li>
    <li>aws-mfa</li>
    <li>aws-sam-cli</li>
    <li>awscli</li>
    <li>awslogs</li>
    <li>barman</li>
    <li>bash-my-aws</li>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
    <li>certbot-full</li>
    <li>checkov</li>
    <li>cloud-custodian</li>
    <li>credstash (python38Packages.credstash)</li>
    <li>deja-dup</li>
    <li>duplicity</li>
    <li>duply</li>
    <li>elasticsearch-curator</li>
    <li>git-remote-codecommit</li>
    <li>iceshelf</li>
    <li>lexicon</li>
    <li>luigi</li>
    <li>mapproxy</li>
    <li>python38Packages.aiobotocore</li>
    <li>python38Packages.aws-sam-translator</li>
    <li>python38Packages.aws-xray-sdk</li>
    <li>python38Packages.boto3</li>
    <li>python38Packages.botocore</li>
    <li>python38Packages.celery</li>
    <li>python38Packages.certbot-dns-route53</li>
    <li>python38Packages.cfn-lint</li>
    <li>python38Packages.chalice</li>
    <li>python38Packages.django-anymail</li>
    <li>python38Packages.django-raster</li>
    <li>python38Packages.djmail</li>
    <li>python38Packages.ec2instanceconnectcli</li>
    <li>python38Packages.fiona</li>
    <li>python38Packages.flowlogs_reader</li>
    <li>python38Packages.fs-s3fs</li>
    <li>python38Packages.gensim</li>
    <li>python38Packages.hass-nabucasa</li>
    <li>python38Packages.kombu</li>
    <li>python38Packages.labelbox</li>
    <li>python38Packages.mapbox</li>
    <li>python38Packages.moto</li>
    <li>python38Packages.pycognito</li>
    <li>python38Packages.pynamodb</li>
    <li>python38Packages.rasterio</li>
    <li>python38Packages.s3fs</li>
    <li>python38Packages.s3transfer</li>
    <li>python38Packages.sagemaker</li>
    <li>python38Packages.sapi-python-client</li>
    <li>python38Packages.scrapy</li>
    <li>python38Packages.scrapy-deltafetch</li>
    <li>python38Packages.scrapy-fake-useragent</li>
    <li>python38Packages.scrapy-splash</li>
    <li>python38Packages.serverlessrepo</li>
    <li>python38Packages.slack-sdk</li>
    <li>python38Packages.slicedimage</li>
    <li>python38Packages.smart_open</li>
    <li>python38Packages.tensorboardx</li>
    <li>python38Packages.warrant</li>
    <li>python39Packages.aiobotocore</li>
    <li>python39Packages.aws-adfs</li>
    <li>python39Packages.aws-sam-translator</li>
    <li>python39Packages.aws-xray-sdk</li>
    <li>python39Packages.boto3</li>
    <li>python39Packages.botocore</li>
    <li>python39Packages.celery</li>
    <li>python39Packages.certbot-dns-route53</li>
    <li>python39Packages.cfn-lint</li>
    <li>python39Packages.chalice</li>
    <li>python39Packages.credstash</li>
    <li>python39Packages.django-anymail</li>
    <li>python39Packages.django-raster</li>
    <li>python39Packages.djmail</li>
    <li>python39Packages.ec2instanceconnectcli</li>
    <li>python39Packages.fiona</li>
    <li>python39Packages.flowlogs_reader</li>
    <li>python39Packages.fs-s3fs</li>
    <li>python39Packages.hass-nabucasa</li>
    <li>python39Packages.kombu</li>
    <li>python39Packages.mapbox</li>
    <li>python39Packages.moto</li>
    <li>python39Packages.pycognito</li>
    <li>python39Packages.pynamodb</li>
    <li>python39Packages.rasterio</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.s3transfer</li>
    <li>python39Packages.sagemaker</li>
    <li>python39Packages.sapi-python-client</li>
    <li>python39Packages.scrapy</li>
    <li>python39Packages.scrapy-deltafetch</li>
    <li>python39Packages.scrapy-fake-useragent</li>
    <li>python39Packages.scrapy-splash</li>
    <li>python39Packages.serverlessrepo</li>
    <li>python39Packages.slack-sdk</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.tensorboardx</li>
    <li>python39Packages.warrant</li>
    <li>s3bro</li>
    <li>s4cmd</li>
    <li>sourcehut.buildsrht</li>
    <li>ssm-session-manager-plugin</li>
    <li>streamlit</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 115431` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
8 packages marked as broken and skipped:
</summary>
<ul>
<li>awsebcli</li>
<li>mlflow-server</li>
<li>python38Packages.flower</li>
<li>python38Packages.geopandas</li>
<li>python38Packages.osmnx</li>
<li>python39Packages.flower</li>
<li>python39Packages.geopandas</li>
<li>python39Packages.osmnx</li>
</ul>
</details>
<details>

  <summary>6 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>dvc-with-remotes: log was empty</li>
<li>python38Packages.cartopy: log was empty</li>
<li>python38Packages.gradient: log was empty</li>
<li>python38Packages.snowflake-connector-python: log was empty</li>
<li>python38Packages.snowflake-sqlalchemy: log was empty</li>
<li>python39Packages.cartopy: log was empty</li>
  </ul>
</details>
<details>
<summary>
114 packages built:
</summary>
<ul>
<li>aws-adfs (python38Packages.aws-adfs)</li>
<li>aws-google-auth</li>
<li>aws-mfa</li>
<li>aws-sam-cli</li>
<li>awscli</li>
<li>awslogs</li>
<li>barman</li>
<li>bash-my-aws</li>
<li>buildbot (python38Packages.buildbot)</li>
<li>buildbot-full (python38Packages.buildbot-full)</li>
<li>buildbot-ui (python38Packages.buildbot-ui)</li>
<li>certbot-full</li>
<li>checkov</li>
<li>cloud-custodian</li>
<li>credstash (python38Packages.credstash)</li>
<li>deja-dup</li>
<li>duplicity</li>
<li>duply</li>
<li>elasticsearch-curator</li>
<li>git-remote-codecommit</li>
<li>home-assistant</li>
<li>iceshelf</li>
<li>lexicon</li>
<li>luigi</li>
<li>mapproxy</li>
<li>python38Packages.aiobotocore</li>
<li>python38Packages.aws-sam-translator</li>
<li>python38Packages.aws-xray-sdk</li>
<li>python38Packages.boto3</li>
<li>python38Packages.botocore</li>
<li>python38Packages.celery</li>
<li>python38Packages.certbot-dns-route53</li>
<li>python38Packages.cfn-lint</li>
<li>python38Packages.chalice</li>
<li>python38Packages.django-anymail</li>
<li>python38Packages.django-raster</li>
<li>python38Packages.djmail</li>
<li>python38Packages.ec2instanceconnectcli</li>
<li>python38Packages.fiona</li>
<li>python38Packages.flowlogs_reader</li>
<li>python38Packages.fs-s3fs</li>
<li>python38Packages.gensim</li>
<li>python38Packages.hass-nabucasa</li>
<li>python38Packages.kombu</li>
<li>python38Packages.labelbox</li>
<li>python38Packages.mapbox</li>
<li>python38Packages.moto</li>
<li>python38Packages.pycognito</li>
<li>python38Packages.pynamodb</li>
<li>python38Packages.rasterio</li>
<li>python38Packages.s3fs</li>
<li>python38Packages.s3transfer</li>
<li>python38Packages.sapi-python-client</li>
<li>python38Packages.scrapy</li>
<li>python38Packages.scrapy-deltafetch</li>
<li>python38Packages.scrapy-fake-useragent</li>
<li>python38Packages.scrapy-splash</li>
<li>python38Packages.serverlessrepo</li>
<li>python38Packages.slack-sdk</li>
<li>python38Packages.slicedimage</li>
<li>python38Packages.smart_open</li>
<li>python38Packages.tensorboardx</li>
<li>python38Packages.warrant</li>
<li>python39Packages.aiobotocore</li>
<li>python39Packages.aws-adfs</li>
<li>python39Packages.aws-sam-translator</li>
<li>python39Packages.aws-xray-sdk</li>
<li>python39Packages.boto3</li>
<li>python39Packages.botocore</li>
<li>python39Packages.celery</li>
<li>python39Packages.certbot-dns-route53</li>
<li>python39Packages.cfn-lint</li>
<li>python39Packages.chalice</li>
<li>python39Packages.credstash</li>
<li>python39Packages.django-anymail</li>
<li>python39Packages.django-raster</li>
<li>python39Packages.djmail</li>
<li>python39Packages.ec2instanceconnectcli</li>
<li>python39Packages.fiona</li>
<li>python39Packages.flowlogs_reader</li>
<li>python39Packages.fs-s3fs</li>
<li>python39Packages.hass-nabucasa</li>
<li>python39Packages.kombu</li>
<li>python39Packages.mapbox</li>
<li>python39Packages.moto</li>
<li>python39Packages.pycognito</li>
<li>python39Packages.pynamodb</li>
<li>python39Packages.rasterio</li>
<li>python39Packages.s3fs</li>
<li>python39Packages.s3transfer</li>
<li>python39Packages.sapi-python-client</li>
<li>python39Packages.scrapy</li>
<li>python39Packages.scrapy-deltafetch</li>
<li>python39Packages.scrapy-fake-useragent</li>
<li>python39Packages.scrapy-splash</li>
<li>python39Packages.serverlessrepo</li>
<li>python39Packages.slack-sdk</li>
<li>python39Packages.slicedimage</li>
<li>python39Packages.tensorboardx</li>
<li>python39Packages.warrant</li>
<li>s3bro</li>
<li>s4cmd</li>
<li>sourcehut.buildsrht</li>
<li>sourcehut.dispatchsrht</li>
<li>sourcehut.gitsrht</li>
<li>sourcehut.hgsrht</li>
<li>sourcehut.hubsrht</li>
<li>sourcehut.listssrht</li>
<li>sourcehut.mansrht</li>
<li>sourcehut.metasrht</li>
<li>sourcehut.pastesrht</li>
<li>sourcehut.todosrht</li>
<li>ssm-session-manager-plugin</li>
<li>streamlit</li>
</ul>
</details>
